AirCasting/Utils/ButtonStyles/GreenButtonStyle.swift

```diff
@@ -0,0 +1,26 @@
+// Created by Lunar on 09/06/2021.
+//
+
+import SwiftUI
+
+struct GreenButtonStyle: ButtonStyle {
+    
+    func makeBody(configuration: Configuration) -> some View {
+        configuration
+            .label
+            .foregroundColor(.white)
+            .frame(maxWidth: .infinity)
+            .padding(.vertical, 12)
+            .background(Color("OnboardingMint"))
+            .cornerRadius(5)
+            .padding(-3)
+            .shadow(color: Color(white: 150/255, opacity: 0.5), radius: 9, x: 0, y: 1)
```

Comment:
how about defining 
`static let shadowColor = Color(white: 150/255, opacity: 0.5)` in Color struct extension